### PR TITLE
7.8.0 release post

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
Feel free to directly edit the contents.

### CODE REVIEW

File: website/blog/2020-01-10-7.8.0.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+date:   2020-01-10 20:00:00
+categories: announcements
+share_text: "Babel 7.8.0 Released"
+---
+
+This is the first release of the year! :tada: 
+
+Babel 7.8.0 supports the new [ECMAScript 2020](https://github.com/tc39/proposals/blob/master/finished-proposals.md) features by default: you don't need to enable individual plugins for nullish coalescing (`??`), optional chaining (`?.`) and dynamic `import()` anymore with preset-env.
+
+We also finished unifying our different configuration formats, a process that we started in the [7.7.0 release](https://babeljs.io/blog/2019/11/05/7.7.0#new-configuration-file-extensions-10501-https-githubcom-babel-babel-pull-10501-10599-https-githubcom-babel-babel-pull-10599).
```
Comment (hzoo) [inline website/blog/2020-01-10-7.8.0.md]: we wanted to use another word other than 'unify' in this case right?

Comment (nicolo-ribaudo) [reply website/blog/2020-01-10-7.8.0.md]: Oh right

Comment (nicolo-ribaudo) [reply website/blog/2020-01-10-7.8.0.md]: Does this look good?

```suggestion
We also finished aligning our different configuration files with the formats natively supported by Node.js, a process that we started in the [7.7.0 release](https://babeljs.io/blog/2019/11/05/7.7.0#new-configuration-file-extensions-10501-https-githubcom-babel-babel-pull-10501-10599-https-githubcom-babel-babel-pull-10599).
```

File: website/blog/2020-01-10-7.8.0.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+<div style="margin: auto;width: fit-content;">
+
+| **Node.js file type** | **babel.config.__** | **.babelrc.__** |
+|:---------------------:|:-------------------:|:------------:|
+| `.js`                 | ✔️ Supported         | ✔️ Supported  |
+| `.json`               | ❌ Not supported    | ❔ Supported, with implicit extension |
+</div>
+
+> ℹ️ We strongly recommend reading about the differences between [`babel.config.*` and `.babelrc.*`](https://babeljs.io/docs/en/config-files)!
```
Comment (nicolo-ribaudo) [inline website/blog/2020-01-10-7.8.0.md]:
```suggestion
> ℹ️ We strongly recommend reading about the differences between [`babel.config.js` and `.babelrc.js`](https://babeljs.io/docs/en/config-files)!
```

**Pull request merged**